File: OOP_Lab/lab3.cpp
#include <iostream>
using namespace std;

class maxandmin
{
public:
    int num1, num2;

    maxandmin(int num1, int num2)
    {
        this->num1 = num1;
        this->num2 = num2;
    }
    void ans()
    {
        if (num1 > num2)
        {
            cout << "Maximum number is: " << num1 << endl;
            cout << "Minimum number is: " << num2 << endl;
        }
        else if (num1 < num2)
        {
            cout << "Maximum number is: " << num2 << endl;
            cout << "Minimum number is: " << num1 << endl;
        }
        else
        {
            cout << "Both numbers are equal: " << num1 << endl;
        }
    }
};

int main()
{
    int a, b;
    cout << "Enter the number of two:" << endl;
    cin >> a >> b;
    maxandmin *m = new maxandmin(a, b);
    m->ans();

    return 0;
}

File: OOP_Lab/lab3.java
import java.util.Scanner;

class MaxAndMin {
    int num1, num2;

    MaxAndMin(int num1, int num2) {
        this.num1 = num1;
        this.num2 = num2;
    }

    void ans() {
        if (num1 > num2) {
            System.out.println("Maximum number is: " + num1);
            System.out.println("Minimum number is: " + num2);
        } else if (num1 < num2) {
            System.out.println("Maximum number is: " + num2);
            System.out.println("Minimum number is: " + num1);
        } else {
            System.out.println("Both numbers are equal: " + num1);
        }
    }
}

public class lab3 {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter two numbers:");
        int a = scanner.nextInt();
        int b = scanner.nextInt();

        MaxAndMin m = new MaxAndMin(a, b);
        m.ans();

    }
}
